package.json

```diff
@@ -46,7 +46,7 @@
     "angular-ui-ace": "^0.2.3",
     "angular-ui-bootstrap": "^2.5.0",
     "angular-vs-repeat": "^1.1.7",
-    "antd": "^3.7.1",
+    "antd": "^3.12.3",
```

Comment:
Note for future me: [link](https://github.com/getredash/redash/pull/2756#issuecomment-453991364).